cmd/common_test.go

```diff
@@ -211,9 +211,9 @@ func TestMapToCISVersion(t *testing.T) {
 		{kubeVersion: "1.13", succeed: true, exp: "cis-1.4"},
 		{kubeVersion: "1.14", succeed: true, exp: "cis-1.4"},
 		{kubeVersion: "1.15", succeed: true, exp: "cis-1.5"},
-		{kubeVersion: "1.16", succeed: true, exp: "cis-1.5"},
-		{kubeVersion: "1.17", succeed: true, exp: "cis-1.5"},
-		{kubeVersion: "1.18", succeed: true, exp: "cis-1.5"},
+		{kubeVersion: "1.16", succeed: true, exp: "cis-1.6"},
+		{kubeVersion: "1.17", succeed: true, exp: "cis-1.6"},
+		{kubeVersion: "1.18", succeed: true, exp: "cis-1.6"},
```

Comment:
We should probably add 1.19 in now since it has just been released (assume it is also covered by cis-1.6)